.github/workflows/validate.yml

```diff
@@ -104,6 +94,9 @@ jobs:
           file: ./.tox/coverage.xml
           name: ${{ matrix.os }}:${{ matrix.python-version }}
           fail_ci_if_error: false
+          # codecov/codecov-action@v3 needs v0.7.3 to work on (intel) macos-13
+          # See https://github.com/codecov/codecov-action/issues/1549
+          version: ${{ matrix.os == 'macos-13' && 'v0.7.3' || 'latest' }}
```

Comment:
I do not understand this statement.
What does it do?

Reply:
It's the equivalent of:

```python
version = matrix.os == "macos-13" and "v042" or "latest"
```
and:
```python
if matrix.os == 'macos-13':
   version = 'v0.7.3'
else:
   version = 'latest'
```